stock_picking_partner_vat/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    partner_vat = fields.Char(related="partner_id.vat")
+    picking_type_code = fields.Selection(related="picking_type_id.code")
```

Comment:
I think this field is already present in the main `stock.picking` definition, so it's unnecessary. Could you check

Reply:
Yes, you are right, it exists and with the same name now I make the change, my fault